HelpMyStreetFE/HelpMyStreetFE/src/js/coveragemap.js

```diff
@@ -399,7 +420,6 @@ function showMarkers() {
 
 function deleteMarkers() {
     clearMarkers();
```

Comment:
deleteMarkers() and clearMarkers() now do exactly the same thing.  I'd suggest removing one of them.